sirepo/package_data/static/json/impactt-schema.json

```diff
@@ -0,0 +1,206 @@
+{
+    "dynamicFiles": {
+        "sirepoLibs": {
+            "js": ["impactt.js"]
+        }
+    },
+    "enum": {
+        "PoleType": [
+            ["sextupole", "Sextupole"],
+            ["octupole", "Octupole"],
+            ["decapole", "Decapole"]
+        ],
+        "DistType": [
+            ["Uniform", "1"],
+            ["Gauss3", "2"],
+            ["Waterbag", "3"],
+            ["Semigauss", "4"],
+            ["KV3d", "5"],
+            ["ParobGauss", "10"],
+            ["SemicircGauss", "15"],
+            ["Read", "16"],
+            ["readParmela", "24"],
+            ["readElegant", "25"],
+            ["CylcoldZSob", "27"],
+            ["Combine", "ijk"]
+        ]
+    },
+    "localRoutes": {
+        "source": {
+            "config": {
+                "controller": "impacttSourceController as source",
+                "templateUrl": "/static/html/ipmactt-source.html"
+            }
+        }
+    },
+    "model": {
+        "simulationStatus": {},
+        "impacttReport": {},
+        "beam": {
```

Comment:
Rename "beam" to "command_beam" so we'll be able to use it on the standard lattice command editor.